routes/topics.js

```diff
@@ -4,15 +4,15 @@ const router = express.Router();
 
 // controller
 const topicsController = require('../controllers/topics');
-const {isAuth, isAuthV2} = require('../middlewares/auth');
+const {isAuth} = require('../middlewares/auth');
 const QueryParamsValidationMiddleware = require('../middlewares/query-params-validation');
 
 /* GET locations listing. */
 router.get('/list', topicsController.topics);
 router.get('/followed', isAuth, topicsController.getFollowedTopic);
 router.get('/follow', isAuth, topicsController.getFollowTopic);
 router.put('/follow', isAuth, topicsController.putFollowTopic);
-router.put('/follow-v2', isAuthV2, topicsController.followTopicV2);
+router.put('/follow-v2', isAuth, topicsController.followTopicV2);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The middleware for the route `/follow-v2` has been changed from `isAuthV2` to `isAuth`. Similar to the above comment, please ensure that `isAuth` provides the necessary functionality and security checks that `isAuthV2` was providing.
```diff
- router.put('/follow-v2', isAuthV2, topicsController.followTopicV2);
+ router.put('/follow-v2', isAuth, topicsController.followTopicV2);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->